var/spack/repos/builtin/packages/py-rpds-py/package.py

```diff
@@ -17,5 +17,8 @@ class PyRpdsPy(PythonPackage):
     license("MIT", checked_by="wdconinc")
 
     version("0.20.0", sha256="d72a210824facfdaf8768cf2d7ca25a042c30320b3020de2fa04640920d4e121")
+    version("0.18.1", sha256="dc48b479d540770c811fbd1eb9ba2bb66951863e448efec2e2c102625328e92f")
 
-    depends_on("py-maturin@1.2:1", type="build")
+    depends_on("rust@1.76:", when="@0.19:")
+    depends_on("py-maturin@1.0:1", type="build")
```

Comment:
FWIW. You should be able to just use `@1` (see https://spack.readthedocs.io/en/latest/packaging_guide.html#ranges-versus-specific-versions).

Reply:
I'm trying to use the same notation as in pyproject.toml, for future reviewers easy of recognizing where this comes originates, i.e. `maturin>=1.0,<2.0`.

Reply:
Understandable.  I can't guarantee I'll remember it though.